setup.cfg

```diff
@@ -61,3 +61,25 @@ ignore =
 
 [devpi:upload]
 formats = sdist.tgz,bdist_wheel
+
+[mypy]
+python_version = 3.5
```

Comment:
I *think* you can drop this since the tests will already fail if invalid syntax is used

Reply:
I'll remove it. I though it'd be nice to have so it already warns about problems in the editor, but CI is still OK I suppose.